tests/test_drone_substeps.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import genesis as gs
+
+
+def test_drone_hover_same_with_and_without_substeps():
+    TOL = 0.02
```

Comment:
This tolerance is quite low. You should be able to get better matching for a simulation lasting 1s.

Reply:
Still need to decrease the tolerance